app.js

```diff
@@ -11,13 +11,15 @@ const Url = require("./models/url");
 
 const app = express();
 
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.json());
+app.use(express.urlencoded({ extended: false }));

```

Comment:
It has been seen that express is not rigid about keeping parser as a default function, so added the body-parser module separately to avoid such issues, Same for .json() as well.